charts/fleet-agent/100.0.0+up0.3.6/Chart.yaml

```diff
@@ -1,8 +1,10 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0-0'
```

Comment:
release in 2.6.0, this looks correct. 

Tested versions are 1.18-1.21 based on the support matrix https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-0, but 1.16-1.17 is fine to keep in the range. 

No changes needed here